strongbox-commons/src/main/java/org/carlspring/strongbox/artifact/coordinates/MavenArtifactCoordinates.java

```diff
@@ -12,9 +12,9 @@
 public class MavenArtifactCoordinates extends AbstractArtifactCoordinates
 {
 
-    private static final String GROUPID = "groupId";
+    private static final String GROUP_ID = "groupId";
```

Comment:
Could you please keep the old names of these variables? I know this doesn't comply with the convention and Sonar is probably complaining. Feel free to ignore this complaint in this case.